admin/config/environment.js

```diff
@@ -38,18 +38,35 @@ module.exports = function (environment) {
       // when it is created
       API_HOST: process.env.API_HOST || '',
       API_ERROR_MESSAGES: {
-        BAD_REQUEST: { CODE: '400', MESSAGE: 'Les données envoyées ne sont pas au bon format.' },
+        BAD_REQUEST: {
+          CODE: '400',
+          MESSAGE: 'api-error-messages.bad-request-error',
```

Comment:
Ici ce n'est pas vraiment le message mais l'identifiant I18N. On pourrait utiliser le nommage suivant

```suggestion
          KEY: 'api-error-messages.bad-request-error',
```

ou

```suggestion
          I18N_KEY: 'api-error-messages.bad-request-error',
```

Reply:
Comme suggéré je viens de modifier les commits pour que le code utilise maintenant `I18N_KEY` à la place de `MESSAGE`.